<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentDialog</class>
 <widget class="QDialog" name="PaymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="startLabel">
     <property name="text">
      <string>Date from</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="startDate"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="stopLabel">
     <property name="text">
      <string>Date to</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="stopDate"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="paymentLabel">
     <property name="text">
      <string>Payment</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="paymentSpin">
     <property name="suffix">
      <string> PLN</string>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="refreshCheck">
       <property name="text">
        <string>Refresh data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PaymentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PaymentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>PaymentDialog</receiver>
   <slot>startDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopDate</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>PaymentDialog</receiver>
   <slot>stopDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startDateChanged(QDate)</slot>
  <slot>stopDateChanged(QDate)</slot>
 </slots>
</ui>
